### DISCUSSION TITLE
Migration from perforce to github.

### DISCUSSION THREAD

**Discussion by sumansrb** [community/community#49986] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

How to migrate code base from perforce to github?
I have got many links of stack overflow, Atlassian and git official documentation but it's not working.
Suggest me the proper method and commands to migrate. I have installed python and set git-p4.py file and p4 commands are working but when I do the git p4 clone depot/path/project command it doesn't work I can only see that an empty git repository is created by the project name.

**Comment by mobingood** [comment]
@sumansrb 

Did you fiound any thing regarding this migration, if yes, can you help me, i too looking for it
